release-1.8/release_notes_draft.md

```diff
@@ -72,6 +72,15 @@ fundamental aspect of a secure cluster.
   * The`--audit-policy-file` requires `kind` and `apiVersion` fields
     specifying what format version the `Policy` is using.
 
+* CronJobs has been promoted to `v1beta1` which is now turned on by default.
+  Although version `v2alpah1` is still available, it is deprecated and users are
```

Comment:
Although version `v2alpah1` is still available, it is deprecated. Migrate to `v1beta1`.

Reply:
Updated.